priv/templates/boot_loader.eex

```diff
@@ -16,7 +16,7 @@ _cleanup() {
     trap - HUP INT TERM QUIT EXIT USR1 USR2
     echo ""
     echo "Shutting down.."
-    "$RELEASES_DIR/$REL_VSN/$REL_NAME.sh" rpc init stop
+    "$RELEASES_DIR/$REL_VSN/$REL_NAME.sh" <%= signal_handler %>
     wait $__pid
```

Comment:
There's definitely a potential that someone will send a command which doesn't shut down the node, and then end up blocking indefinitely because of the `wait`. We don't have a way to handle different signals in different ways here, and one would not be easy to add, unless we go the route of implementing our own version of the OTP 20 signal handler to tide things over. I'm somewhat leaning towards the latter, because realistically I would like to avoid having Distillery be making any of these decisions, and the migration path to the built-in signal handler will be less painful in the future.

Thoughts?